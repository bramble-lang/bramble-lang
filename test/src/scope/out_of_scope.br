fn my_main() {
    let x: i64 := if (true) {
        let y: i64 := 5;
        y
    } else {
        y
    };
    return;
}