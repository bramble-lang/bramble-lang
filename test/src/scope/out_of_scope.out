Error: Semantic: L6: Variable y not declared
